docs/customizing.md

```diff
@@ -316,18 +316,19 @@ Elemental Teal leverages container images to build its root filesystems; therefo
 to use it in a multi-stage environment to create custom bootable media that bundles a custom container image.
 
 ```docker showLineNumbers
-FROM registry.suse.com/rancher/elemental-teal/5.4:latest as os
+FROM registry.suse.com/suse/sle-micro/5.5:latest AS os
 
 # Check the section on remastering a custom docker image
 
-FROM registry.suse.com/rancher/elemental-builder-image/5.4:latest AS builder
+# The released OS already includes the toolchain for building ISOs
+FROM registry.suse.com/suse/sle-micro/5.5:latest AS builder
 
 ARG TARGETARCH
 WORKDIR /iso
 COPY --from=os / rootfs
 
 # work around buildah issue: https://github.com/containers/buildah/issues/4242
-RUN rm rootfs/etc/resolv.conf
+RUN rm -f rootfs/etc/resolv.conf
```

Comment:
On my TW env there was no `rootfs/etc/resolv.conf` file, hence I wonder to what extend this is likely not to be an issue anymore in up to date `buildah` versions. I did not investigate further if this is still needed on older versions of buildah, hence I left it here just in case with the `-f` to prevent the failure in case the file is not there.